tests/setup_machine.sh

```diff
@@ -224,6 +228,11 @@ python_packages=(
 # determine which OS we are running and choose the right set of packages to be installed
 release="$(lsb_release -si)/$(lsb_release -sr)"
 case "${release}" in
+	Ubuntu/24.04)
+		echo $release
+		apt-get -y install ca-certificates-java # https://www.mail-archive.com/debian-bugs-dist@lists.debian.org/msg1911078.html
+		apt-get -y install "${common_packages[@]}" "${apt_packages[@]}" "${noble_packages[@]}"
```

Comment:
According to this, prometheus support will only be available in Ubuntu 24.04?

Reply:
> According to this, prometheus support will only be available in Ubuntu 24.04?

Yes, at least for now.

> It seems that installing on 22.04 is not straightforward https://www.cherryservers.com/blog/install-prometheus-ubuntu

You can still manually compile and have CMake/the executable detect the library (like I did on Arch, which also doesn't have this package), but I can't work on the assumption it might be there.

One idea I had was to download it manually with CMake, but it's a shared library, so it's not going to be easy to setup anyway. Another option is to provide the package in our apt repository.

Reply:
I understand, but at this point, looks like we are dropping support for master features on Ubuntu 22.04.

Reply:
They can still have the feature, they will just have to install the library and build the packages manually. Our (and other) 22.04 packages simply won't be able to provide the package with Prometheus support unless we figure out a way to provide the library ourselves.